cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -204,6 +221,10 @@ private void generateShardConfig() {
     } catch (Exception e) {
       LOG.error("Failed to post the new config", e);
     }
+
+    final long endNanos = System.nanoTime();
+    final int elapsedMicros = (int) ((endNanos - startNanos) / 1000);
+    monitor.reportConfigGeneratorLatency(elapsedMicros);
   }
```

Comment:
Can you encapsulate this in separate StopWatch class...

StopWatch stopWatch = new StopWatch(true) <- start the timer


monitor.reportConfigGeneratorLatency(stopWatch.getElapsedMicros())

Reply:
Great. Thank, StopWatch is better.